src/core/Cline.ts

```diff
@@ -1104,7 +1104,7 @@ export class Cline {
 							return `[${block.name} for '${block.params.path}']`
 						case "write_to_file":
```

Comment:
If we're making this change, I wonder if we should also change `write_to_file` to `create_file`. Seems like a more obvious pairing of tool names.